tsl/test/expected/compression.out

```diff
@@ -1363,13 +1363,116 @@ SELECT reloptions FROM pg_class WHERE relname = :statchunk;
 (1 row)
 
 DROP TABLE stattest;
+--- Test that analyze on compression internal table updates stats on original chunks
+CREATE TABLE stattest2(time TIMESTAMPTZ NOT NULL, c1 int, c2 int);
+SELECT create_hypertable('stattest2', 'time', chunk_time_interval=>'1 day'::interval);
+    create_hypertable    
+-------------------------
+ (29,public,stattest2,t)
+(1 row)
+
+ALTER TABLE stattest2 SET (timescaledb.compress, timescaledb.compress_segmentby='c1');
+INSERT INTO stattest2 SELECT '2020/06/20 01:00'::TIMESTAMPTZ ,1 , generate_series(1, 200, 1);
+INSERT INTO stattest2 SELECT '2020/07/20 01:00'::TIMESTAMPTZ ,1 , generate_series(1, 200, 1);
+SELECT  compress_chunk(ch1.schema_name|| '.' || ch1.table_name)
+FROM _timescaledb_catalog.chunk ch1, _timescaledb_catalog.hypertable ht 
+WHERE ch1.hypertable_id = ht.id and ht.table_name like 'stattest2'
+ ORDER BY ch1.id limit 1;
+              compress_chunk              
+------------------------------------------
+ _timescaledb_internal._hyper_29_58_chunk
+(1 row)
+
+SELECT relname, reltuples, relpages, relallvisible FROM pg_class
+ WHERE relname in ( SELECT ch.table_name FROM 
+                   _timescaledb_catalog.chunk ch, _timescaledb_catalog.hypertable ht
+  WHERE ht.table_name = 'stattest2' AND ch.hypertable_id = ht.id )
+order by relname;
+      relname       | reltuples | relpages | relallvisible 
+--------------------+-----------+----------+---------------
+ _hyper_29_58_chunk |       200 |        2 |             0
+ _hyper_29_59_chunk |         0 |        0 |             0
+(2 rows)
+
+\c :TEST_DBNAME :ROLE_SUPERUSER
+--overwrite pg_class stats for the compressed chunk.
+UPDATE pg_class
+SET reltuples = 0, relpages = 0
```

Comment:
Is there a natural way that the value will be overwritten? Can it be overwritten to a number other than `0`?

Reply:
yes,  it would be 0 , if you are on a db from pre 2.0